### ISSUE TITLE
Arduino is restarting after connects to gateway

### ISSUE THREAD

**Issue by SchmidtDawid** [tobof/openhab2-addons#136]
Arduino is restarting after connects to gateway. It means if I restart openhab, or even update gateway thing settings. Arduino is restarting. How to keep arduino on?

I tried all setting on and off but i nothing changes.

**Comment by tobof** [comment]
Are you using a serial gateway?
In that case this behaviour is intended by the MySensors library, issn't it? The binding only supports a hard reset via DTR line (if supported) and this is disabled by default.
Why is that a problem?

**Comment by SchmidtDawid** [comment]
It is a problem, becouse i have to connect computers to relays. If someday server or OH service will restart by any reason, my copmuters will turn off too. I'll try to avoid it in future. 
Yes i am using Serial Gateway. I disabled all options with reseting. Even turn off sanity check but it changes nothing. 

I tried fing soultion on Mysensors forum but it seeme they don,t know how to help too ;). I wrote all my config there:
https://forum.mysensors.org/topic/10791/loosing-relays-state-after-oh-restart/3
